.eslintrc

```diff
@@ -1,5 +1,5 @@
 {
-  "parser": "@babel/eslint-parser",
+  "parser": "esprima",
```

Comment:
Can we use the default eslint parser (`Espree`)?

Reply:
I mixed up their names, I'll delete this line to use the default `Espree` parser.